UnrealPlugin/ThirdPersonMPCloudDeploy.md

```diff
@@ -15,28 +15,81 @@ This guide explains how to deploy your game to PlayFab Multiplayer Server (MPS)
+* Get familiar with the [build creation process guide](https://docs.microsoft.com/en-us/gaming/playfab/features/multiplayer/servers/deploying-playfab-multiplayer-server-builds)
+* Specific details for specific configurations are detailed below:
 
 ### Setting up a Windows Dedicated Server on PlayFab
 
-An important note about this is that you need to set the start command in process-based mode to \<root folder\>\\Binaries\\Win64\\\<project name\>Server.exe (or however the executable is called in the folder).
-
-In container-based mode use \<mount folder\>\\\<root folder\>\\Binaries\\Win64\\\<project name\>Server.exe (or however the executable is called in the folder).
-
-**If you use the executable in the root folder, the server will fail to initialize with the PlayFab system.**
+This section will cover creating a build in PlayFab MPS, uploading your game server, and connecting to it for the first time.
+
+* Navigate to Game Manager -> Multiplayer
+* Click "New Build"
+* Give the build a name
+* Select a Virtual Machine (For the samples described in these guides, the recommended option with the fewest cores is sufficient)
+* Servers per machine can be aproximately 1~10
+	* Improving/Raising this number is based on the performance of your game server
+* Skip "Server details" (covered in the next step)
+* In the Assets block, drag your zip file from the previous section into the upload box
+	* If you're re-launching, you can choose "Use existing assets"
+* Wait for the upload to complete
+* Skip "Start Command" (covered in the next step)
+* In the Network block, set the Port Name field to ```UnrealServerGsdkHostPort```
+	* It's mandatory to define the port that will be used to connect to your game server
+	* Unreal GSDK contains code to read this port automatically, and internaly override the Unreal Server host port according to this value
+	* It requires that the name of this port be set properly to ```UnrealServerGsdkHostPort```
+* Continue to the next step that matches the "Server Details" that you want to configure
+
+#### Option 1: Windows Process Mode
+
+This option runs your game server(s) as a process directly on the VM. Your processes will share access to the full machine, including ports, drives, etc. This option unzips your asset upload, and executes the "Start Command" from the location of your unzipped files as the working directory.
+
+* In "Server Details", choose "Process" and "Windows"
+* In "Assets", set the start command to:
+	* ```{ProjectName}Server.exe``` or ```{ProjectName}Server.exe -log``` if you want to generate log files
+	* The start command is executed from the command line, using your unzipped asset files as the working directory
+	* If you chose to pack your zip file with sub-folders, you can specify a relative path such as: ```WindowsServer/{ProjectName}Server.exe``` to access your project with a relative sub-path
+* Click "Add Build"
```

Comment:
Use "Select" instead of "Click"

Usually we bold UI text. So it'll be __Add Build __. Applies to the rest too.